_component_design/time-selector.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Time Selector
+parent: Inputs and Controls
+layout: component
+category: Components
+usage: The time selector enables the user to select a time from a list of values. The time selector is often accompanied by a date selector that a user can use to select a date.
+preview-image: preview-images/time-selector.svg
+resource: true
+status: stable
+last-modified: 2018-11-13
+helix-ui-css: false
+helix-ui-javascript: false
+---
+
+{% include toc.html %}
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Use the time selector when you want to enable a user to select a time from a list of values. The time selected includes an hour value and a minute value.
+
+If you want the user to select a time from a limited list of predetermined times, consider using [radios]({{site.baseurl}}\components\radios.html).
```

Comment:
Our logic is a little muddled in the `When to Use` section.  The first sentence says that the selector should be used when you want the user to select from a list of values, but then the third sentence says that if you want the user to select a time from a limited list of predetermined times, to consider using radios.

If we don't want implementors to use the time selector for a set of limited, predetermined values, then we should be explicit in the first sentence and say something along the lines of  `Use the time selector to enable a user to choose a specific hour in the 24-hour range.`